metaflow/tutorials/09-hellokfp/parameter_flow.py

```diff
@@ -0,0 +1,46 @@
+from metaflow import FlowSpec, Parameter, step
+
+
+class ParameterFlow(FlowSpec):
+    """
+    A flow where Metaflow prints 'Hi'.
+
+    The hello step uses @resource decorator that only works when kfp plug-in is used.
+    """
+
+    alpha = Parameter(
+        'alpha',
+        help='param with default',
+        default=0.01,
+    )
+
+    beta = Parameter(
+        'beta',
+        help='param with no default',
+        type=int,
+        required=True
+    )
+
+    def __init__(self):
+        super(ParameterFlow, self).__init__()
+
```

Comment:
Is this constructor necessary?

Do we need another sample and not instead rely on the existing metaflow example? 

How about adding a flow parameter and test to our kfp plugin tests?

Reply:
Will work on adding an automated test later this week.